kong/db/migrations/core/024_380_to_390.lua

```diff
@@ -14,6 +14,9 @@ return {
         "entity" JSON,
         FOREIGN KEY (version) REFERENCES clustering_sync_version(version) ON DELETE CASCADE
       );
+      -- version starts at 1, as 0 indicates no sync has been performed
+      INSERT INTO clustering_sync_version (version) VALUES (1) ON CONFLICT DO NOTHING;
+      INSERT INTO clustering_sync_delta (version, type, pk, ws_id, entity) VALUES (1, 'init', '{}', '00000000-0000-0000-0000-000000000000', '{}') ON CONFLICT DO NOTHING;
```

Comment:
I think that `type` should be `routes` `services` or others, it could not be `init`.